src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -875,6 +875,7 @@ Object AsyncRPCOperation_sendmany::perform_joinsplit(
             {info.vjsout[0], info.vjsout[1]};
     boost::array<size_t, ZC_NUM_JS_INPUTS> inputMap;
     boost::array<size_t, ZC_NUM_JS_OUTPUTS> outputMap;
+    std::function<int(int)> emptyFunc;
```

Comment:
This is why I suggested placing it in `src/random.h`. It's an identity function relative to the shuffle being used (a Durstenfeld shuffle). The shuffle iterates backwards over the array, swapping the pointed-at element with the element selected by `gen` (from 0 to the pointer). So by always outputting `n-1`, gen always selects the pointer itself, meaning that the shuffle only swaps elements with themselves, making it a no-op or identity shuffle.